source/documentation/deploying-an-app/use-circleci-to-upgrade-app.html.md.erb

```diff
@@ -8,7 +8,7 @@ review_in: 3 months
 
 
 ## Introduction
-This document covers how to continuously deploy your application in the Cloud Platform. It is essentially a continuation of [‘Deploying an application to the Cloud Platform with Helm’][deploy-helm].
+This document covers how to continuously deploy your application in the Cloud Platform. 
 
 *Note: This document is specific to using [CircleCI](https://circleci.com/) as the Continuous Integration method.*
```

Comment:
I'd recommend using `>` here to call out notes, rather than bracketing with `*` e.g.

> Note: This document...

Reply:
done